exasol_integration_test_docker_environment/test/test_ssh_access.py

```diff
@@ -1,11 +1,18 @@
+import os
 import pytest
 
-from exasol_integration_test_docker_environment.lib.base.ssh_access import SshKey
+from .conftest import find_container
+from exasol_integration_test_docker_environment.lib.base.ssh_access import SshFiles
 
 
 def test_generate_ssh_key_file(database):
+    test_name = "test_generate_ssh_key_file"
     with database(
```

Comment:
Depending on what you want to test, you should to prefer the ApiTestEnvironment. The fixtures' cli_isolation and database (we should rename to database_via_cli) should only be used, if you need to test the CLI, if you intend to test the actual logic you should ApiTestEnvironment and maybe create similar fixtures around it. Reason, is with ApiTestEnvironment you can debug and don't run in a container.

Reply:
Switched to ApiTestEnvironment